clients/py/v3io_frames/http.py

```diff
@@ -152,13 +152,18 @@ def _delete(self, backend, table, filter, start, end, if_missing, metrics):
             'session': pb2py(self.session),
             'backend': backend,
             'table': table,
-            'filter': filter,
-            'start': start,
-            'end': end,
             'if_missing': if_missing,
-            'metrics': metrics,
         }
 
+        if filter:
```

Comment:
Just change the default value of `Metrics` in the delete function definition in client.py:342
the filter,start,end change is redundant